packages/react/src/returns/hooks/types/useUserReturns.ts

```diff
@@ -1,7 +1,13 @@
-import type { Config, GetUserReturnsQuery } from '@farfetch/blackout-client';
+import type {
+  Config,
+  GetUserReturnsLegacyData,
+  GetUserReturnsQuery,
+} from '@farfetch/blackout-client';
 
 export type UseUserReturnsOptions = {
   enableAutoFetch?: boolean;
   fetchConfig?: Config;
   fetchQuery?: GetUserReturnsQuery;
```

Comment:
```suggestion
  fetchQuery?: GetUserReturnsQuery | GetUserReturnsLegacyQuery;
```

Reply:
Actually scratch that, I see that you are using the `orderId` parameter from `GetUserReturnsQuery` which does not exist in `GetUserReturnsLegacyQuery`.